source/_patterns/03-templates/page-grid/page-grid.twig

```diff
@@ -0,0 +1,32 @@
+{% set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['grid']) }) -%}
+
```

Comment:
This whole file will probably need replacing. With what is yet to be determined.